include/sys_clock.h

```diff
@@ -130,6 +206,8 @@ s64_t z_tick_get(void);
 #define z_tick_get_32() (0)
 #endif
 
+u64_t z_timeout_end_calc(k_timeout_t timeout);
```

Comment:
what does this function do? Missing dox.

Reply:
There's a big comment above it's implementation describing what it does, with a warning about how to use it correctly.  In general we don't doxygen internal functions.  It's declared here only so it can be used in an inline.

Reply:
ok, i found the comment in the implementation. Imo, such comments should be in the header even though these functions are not public.

Reply:
Yeah, I think we have to agree to disagree on this one.  I really don't think putting documentation for internal functions into public headers is a good idea.